### PR TITLE
Made json chages and aded toogle functionality

### PR DESCRIPTION
**Pull request by satishshewaleglobant**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -13,48 +13,73 @@ class Dashboard extends Component {
...
     return (
-      <button onClick={() => this.handleopenModal(row,'GK')} id="btnopenModel">
+      <button className="btn" onClick={() => this.handleopenModal(row, 'GK')} id="btnopenModelGK">
         GK
       </button>
     );
   }
+  dataToDashboard = (recivedObjData) => {
+    this.setState({
+      formdata: [...this.state.list, recivedObjData]
```
Comment (satishshewaleglobant) [inline src/components/Dashboard/Dashboard.js]: I dont think we need form data in state

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -13,48 +13,73 @@ class Dashboard extends Component {
...
+    });
+    console.log("recivedObjData", recivedObjData);
+    let updateRowData = this.state.list.find(filterData => filterData.id === recivedObjData.id);
```
Comment (satishshewaleglobant) [inline src/components/Dashboard/Dashboard.js]: Can we use index to update data instead traversing in array?

File: src/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -13,48 +13,73 @@ class Dashboard extends Component {
...
+    updateRowData.evaluateresult = recivedObjData.evaluateresult;
+    updateRowData.seniority =  recivedObjData.seniority;
+    updateRowData.feedback= recivedObjData.feedback;
+   // console.log("updateRowData", updateRowData);  
 
+    let elementL1 = document.getElementById('btnopenModelL1'); 
+    let elementGK = document.getElementById('btnopenModelGK');
+
+    if(updateRowData.evaluateresult=="Selected"){
+    //  alert("1");
```
Comment (satishshewaleglobant) [inline src/components/Dashboard/Dashboard.js]: I think this should go in html

Comment (mugdhasonawane88) [reply src/components/Dashboard/Dashboard.js]: Removed this code from the file

File: src/components/Dashboard/Dashboard.js (hunk 1 above)
Comment (mugdhasonawane88) [reply src/components/Dashboard/Dashboard.js]: this data is no longer in use so removing out those lines of code

**Pull request merged**